website/source/docs/enterprise/index.html.md

```diff
@@ -29,7 +32,21 @@ Enterprise](https://www.hashicorp.com/consul.html).
 Licensing capabilities were added to Consul Enterprise v1.1.0. The license is set
 once for a datacenter and will automatically propagate to all nodes within the
 datacenter over a period of time scaled between 1 and 20 minutes depending on the
-number of nodes in the datacenter. The license can be set via the 
+number of nodes in the datacenter. There are two methods for licensing Consul
+enterprise.
+
+### Included in the Enterprise Package
+
+If you are downloading Consul from Amazon S3, then the license is included
+and you do not need to take further action. This is the most common use 
+case.
+
+### Applied after Bootstrapping
+
+If you are downloading the enterprise binary from the [public hub](https://releases.hashicorp.com/consul/), you will need to apply
+the license to the leading server after bootstrapping the cluster. 
```

Comment:
You can set the license on any enterprise agent connected in the cluster and the request gets forwarded to the leader to be applied. So you can send it to either a client, non-leader server or the leader and it will work the same. Maybe we want to simplify things and just say to do it against the leader. I am fine either way.